internal/test_other.go

```diff
@@ -10,7 +12,7 @@ import (
 	"github.com/google/go-tpm-tools/simulator"
 )
 
-var tpmPath = flag.String("tpm_path", "", "Path to Linux TPM character device (i.e. /dev/tpm0 or /dev/tpmrm0). Empty value (default) will run tests against the simulator.")
+var tpmPath = flag.String("tpm-path", "", "Path to Linux TPM character device (i.e. /dev/tpm0 or /dev/tpmrm0). Empty value (default) will run tests against the simulator.")
```

Comment:
Are these packages only part of the build for tests? Otherwise I would be concerned with the flag potentially conflicting with a real flag by the users' binary.

Reply:
This flag is only used for testing. I added some comments as to why this is the case, and how the conflict is avoided.